src/shared/components/CloudUpgradeButton.tsx

```diff
@@ -49,6 +49,7 @@ const CloudUpgradeButton: FC<StateProps & OwnProps> = ({
     [`${className}`]: className,
   })
 
+  // TODO(ariel): update the link to the local checkout if the flag is on
```

Comment:
looks like you did this in `CloudUpgradeNavBanner`? Or is that a different component?

Reply:
Different component. This one is dependent on the `LinkButton` component which takes in an `href` rather than a `to` to link to a page. I've got an issue in clockface for it here:

https://github.com/influxdata/clockface/issues/623

Reply:
JK i'm dumb. Closing that issue. Just made it a button